src/daemon/daemon.cpp

```diff
@@ -1504,6 +1594,38 @@ catch (const std::exception& e)
     return grpc::Status(grpc::StatusCode::FAILED_PRECONDITION, e.what(), "");
 }
 
+grpc::Status mp::Daemon::restart(grpc::ServerContext* context, const RestartRequest* request,
+                                 grpc::ServerWriter<RestartReply>* server) // clang-format off
+try
+{
+    mpl::ClientLogger<RestartReply> logger{mpl::level_from(request->verbosity_level()), *config->logger, server};
+
+    auto tgts_and_st = process_targets(request->instance_name(), vm_instances, deleted_instances);
+    const auto& tgts = tgts_and_st.first; // use structured bindings instead in C++17
```

Comment:
Same for variable name here.  `instances` or something descriptive.

Reply:
Done.